hyou/view.py

```diff
@@ -37,6 +43,11 @@ def __init__(self, worksheet, api, start_row, end_row, start_col, end_col):
         self._input_value_map = {}
         self._cells_fetched = False
         self._queued_updates = []
+        self._major_dimension = kwargs.get(
+            'major_dimension', DEFAULT_MAJOR_DIMENSION
+        )
+        self._value_type = kwargs.get('value_type', DEFAULT_VALUE_TYPE)
+        self._date_type = kwargs.get('date_type', DEFAULT_DATE_TYPE)
```

Comment:
Could the other possible values for the three new dimensions be documented? Maybe have them as constants somewhere. Right now it's difficult to know what this arguments do or what values they can have 😝 Maybe a link to the official docs would be enough.